acceptance/config/vmpooler-el7-64mda.cfg

```diff
@@ -0,0 +1,14 @@
+HOSTS:
+  vmpooler-el7-mda:
+    roles:
+      - master
+      - agent
+      - database
+    platform: el-7-x86_64
+    hypervisor: vcloud
+    template: redhat-7-x86_64
+CONFIG:
+  datastore: instance0
+  resourcepool: delivery/Quality Assurance/FOSS/Dynamic
+  folder: Delivery/Quality Assurance/FOSS/Dynamic
+  pooling_api: http://vmpooler.delivery.puppetlabs.net/
```

Comment:
@kbarber I have been using this as a bootstrapping mechanism.

@justinstoller if this config is not needed for getting the Jenkins running, please let me know and I will remove it.